### PR TITLE
[dv-processing] Update to latest version

### PR DESCRIPTION
**Pull request by jureviciusr**
**Describe the pull request**

- #### What does your PR fix?
Update to newly released version 1.5.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
all

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
yes

### CODE REVIEW

File: ports/dv-processing/portfile.cmake [hunk 1]
```diff
@@ -2,11 +2,9 @@ vcpkg_from_gitlab(
         GITLAB_URL https://gitlab.com/inivation
         OUT_SOURCE_PATH SOURCE_PATH
         REPO dv/dv-processing
-        REF rel_1.4
-        SHA512 c011ca0e6d9842913ff35b0a03f9053bfbc98c090b6936e01f6514b8a35d31ee6d0a821f491be96400113e93967aa2d3e8ab19e558f5c3e9f8eba9ad4e1fe013
-        HEAD_REF d4ffab46a2849372789c5a2084821011165086ab
-        PATCHES
-                vcpkg-build.patch
+        REF 96d082a862bb1e5bfdc79b39aa09e7a50c2dac49
```
Comment (JonLiu1993) [inline ports/dv-processing/portfile.cmake:5]: @jureviciusr, Thanks for your pr, currently the latest stable version is 1.5.0 and the commit id is 
9b1d8bb2e2223f76a4b1f12ae3998933947855b3, could you please change the REF?

File: ports/dv-processing/vcpkg.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "dv-processing",
-  "version": "1.4.0",
-  "port-version": 1,
+  "version": "1.5.0",
+  "port-version": 2,
```
Comment (JonLiu1993) [inline ports/dv-processing/vcpkg.json:4]:
```suggestion
```

Comment (JonLiu1993) [reply ports/dv-processing/vcpkg.json:4]: When we update port, we should remove the 'port version'

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/dv-processing/portfile.cmake (hunk 1 above)
Comment (jureviciusr) [reply ports/dv-processing/portfile.cmake:5]: The commit 9b1d8bb2e2223f76a4b1f12ae3998933947855b3 points to release version of 1.5.0, but a few days later additional hot-fix was introduced and the head commit is 96d082a862bb1e5bfdc79b39aa09e7a50c2dac49 now, you can see that here: https://gitlab.com/inivation/dv/dv-processing/-/commits/rel_1.5

**Review by JonLiu1993 — APPROVED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thanks!

**Pull request merged**